component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -484,7 +484,25 @@ private void processSMSOTPFlow(AuthenticationContext context, HttpServletRequest
                 String loginPage = SMSOTPUtils.getMobileNumberRequestPage(context);
                 try {
                     String url = getURL(loginPage, queryParams);
-                    response.sendRedirect(url);
+                    String mobileNumberPatternViolationIssue = SMSOTPConstants.MOBILE_NUMBER_PATTERN_POLICY_VIOLATED;
+                    if (StringUtils.isNotEmpty(
```

Comment:
```suggestion
                    String  moblieNumberRegex = context.getAuthenticatorProperties().get(SMSOTPConstants.MOBILE_NUMBER_REGEX)
                    if (StringUtils.isNotEmpty(moblieNumberRegex)){ 
```

Reply:
Shall we extract the reused AuthenticatorProperties into a string and use that string, it is more readable, WDYT?